### PR TITLE
MIGENG-1001: Select migration network

### PR DESCRIPTION
**Pull request by apinnick**
https://issues.redhat.com/browse/MIGENG-1001

Doc previews:

UI: https://deploy-preview-81--distracted-shaw-5cd1c3.netlify.app/documentation/doc-migration_toolkit_for_virtualization/master/#selecting-migration-network_forklift

CLI: https://deploy-preview-81--distracted-shaw-5cd1c3.netlify.app/documentation/doc-migration_toolkit_for_virtualization/master/#migrating-virtual-machines-cli_forklift (Host CR)

### CODE REVIEW

**Comment by apinnick** [comment]
Waiting for info about CLI from Fabien.

File: documentation/doc-Migration_Toolkit_for_Virtualization/modules/migrating-virtual-machines-cli.adoc [hunk 1]
```diff
@@ -110,28 +110,29 @@ spec:
...
 <1> Specify the name of the `Plan` CR.
 <2> You can create multiple network mappings for source and destination networks.
 <3> You can use either the `id` _or_ the `name` parameter to specify the source network.
 <4> _Managed object reference_ of the source network.
-<5> You can create multiple storage mappings for source data stores and destination storage classes.
-<6> You can use either the `id` _or_ the `name` parameter to specify the source data store.
-<7> _Managed object reference_ of the source data store.
-<8>  You can use either the `id` _or_ the `name` parameter to specify the source VM.
-<9> _Managed object reference_ of the source VM.
+<5> Specify the name of the default migration transfer network. The default is the `pod` network, which is not optimal for disk transfer.
```
Comment (fdupont-redhat) [inline documentation/doc-Migration_Toolkit_for_Virtualization/modules/migrating-virtual-machines-cli.adoc:130]: Here, we're configuring the network mapping, which has nothing to do with the migration network. I wouldn't change this.

@mansam what attribute is set to specify the migration network?

File: documentation/doc-Migration_Toolkit_for_Virtualization/modules/selecting-migration-network-for-target-provider.adoc [hunk 2]
```diff
@@ -0,0 +1,16 @@
+// Module included in the following assemblies:
+//
+// * documentation/doc-Migration_Toolkit_for_Virtualization/master.adoc
+
+[id="selecting-migration-network_{context}"]
+= Selecting a migration network for {a-virt} provider
+
+You can select a default migration network for {a-virt} provider in the {project-short} web console to improve performance. The default migration network is the `pod` network, which is not optimal for disk transfer.
```
Comment (fdupont-redhat) [inline documentation/doc-Migration_Toolkit_for_Virtualization/modules/selecting-migration-network-for-target-provider.adoc:8]: "which **may** not be optimal for disk transfer"

We don't know if the pod network is fine or not.

File: documentation/doc-Migration_Toolkit_for_Virtualization/modules/migrating-virtual-machines-cli.adoc (hunk 1 above)
Comment (apinnick) [reply documentation/doc-Migration_Toolkit_for_Virtualization/modules/migrating-virtual-machines-cli.adoc:130]: OK. I will leave this comment unresolved but change the text.

Comment (apinnick) [reply documentation/doc-Migration_Toolkit_for_Virtualization/modules/migrating-virtual-machines-cli.adoc]: @mansam Still waiting for your input.
@fdupont-redhat Do you have any input for selecting OCP network from the CLI?

**Pull request closed without merging**